fuzz/fuzz_test.go

```diff
@@ -0,0 +1,15 @@
+package fuzz
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/suyashkumar/dicom"
+)
+
+func Fuzz(f *testing.F) {
```

Comment:
Can we rename this to specify what's being Fuzzed?
```suggestion
func FuzzDICOMParse(f *testing.F) {
```

Reply:
Done